manifests/deploy.pp

```diff
@@ -40,8 +41,9 @@
     controller   => $controller,
     ctrluser     => $jboss::internal::runtime::node::username,
     ctrlpasswd   => $jboss::internal::runtime::node::password,
+    runtime_name => $runtime_name,
```

Comment:
Validate runtime name parameter to ensure that runtime name ends with `.ear`, `.rar`, `.jar` or `.war`